update-cloudformation-template

```diff
@@ -131,6 +131,7 @@ function generate_templates() {
                    "ap-northeast-1"
                    "ap-northeast-2"
                    # "ap-northeast-3" #  Disabled for now because we do not have access
+                   "af-south-1"
```

Comment:
Could you also maybe add "ap-southeast-3" and "eu-south-1". Those seem to be missing too.

Reply:
I took those from this list: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-regions

Reply:
Done here as well!